configure.ac

```diff
@@ -364,7 +364,7 @@ m4_pattern_forbid([^_?PKG_[A-Z_]+$],[*** pkg.m4 missing, please install pkg-conf
 have_dbus=no
 AC_ARG_ENABLE(dbus, AS_HELP_STRING([--disable-dbus], [disable usage of dbus-1 in tests]))
 AS_IF([test "x$enable_dbus" != "xno"], [
-        PKG_CHECK_MODULES(DBUS, [dbus-1 >= 1.3.2],
+        PKG_CHECK_MODULES(DBUS, [dbus-1 >= 1.9.18],
```

Comment:
oh, good point... we only link against libdbus for tests these days, hence bumping the dependency for that doesn't make too much sense i figure... but also doesn't hurt... dunno...